src/require.ts

```diff
@@ -17,7 +17,7 @@ export function jitiRequire(
 
   // Check for node:, file:, and data: protocols
   if (id.startsWith("node:")) {
-    id = id.slice(5);
+    return nativeImportOrRequire(ctx, id, opts.async);
```

Comment:
When using `node:` protocol, `import`/`require` will fail on an unknown module:
```
node:internal/modules/esm/translators:358
    throw new ERR_UNKNOWN_BUILTIN_MODULE(url);
          ^

Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:x
    at ModuleLoader.builtinStrategy (node:internal/modules/esm/translators:358:11)
    at #translate (node:internal/modules/esm/loader:534:12)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:581:27) {
  code: 'ERR_UNKNOWN_BUILTIN_MODULE'
}
```

That is why it probably does not make sense to run the `isBuiltin()` check: userland modules cannot use the `node:` protocol anyway.